code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -16,7 +16,6 @@
 	icon_living = "drone_synd"
 	picked = TRUE //the appearence of syndrones is static, you don't get to change it.
 	health = 30
-	maxHealth = 120 //If you murder other drones and cannibalize them you can get much stronger
```

Comment:
this should remain, but you should set `bruteloss = 90` below it

Reply:
That wouldn't stop the syndrone from just screwdrivering itself up to full health after it spawns.

The real solution would be to have it start with 30 maxHealth and raise that when it cannibalizes drones.

Reply:
999 health for master eaters